tests/server.test.js

```diff
@@ -0,0 +1,52 @@
+const { MongoMemoryServer } = require('mongodb-memory-server');
+const supertest = require('supertest');
+const initByDbConn = require('../server/scripts/db.init_by_dbconn');
+const createApp = require('../server/app');
+
+let app;
+let db;
+
+describe('Test /api/ path', () => {
+  beforeAll(async () => {
+    db = await MongoMemoryServer.create();
+    const uri = db.getUri();
+    await initByDbConn(uri);
+    app = await createApp(uri);
+  });
+
+  test('Categories array contain required categories count', async () => {
+    await supertest(app)
```

Comment:
@IgorChaikin You can wrap your server in `supertest` only once, instead of doing it in each test case

```js
let agent

beforeAll(async () => {
  agent = supertest(app);
})

describe('', () => {})

describe('', () => {})

afterAll(async () => {
  await db.stop();
});
```